### ISSUE TITLE
ncolumn does not exist. torchvision.utils.make_grid(normal_maps_plot, scale_each=False, normalize=False, ncolumn=plot_nrow)

### ISSUE THREAD

**Issue by volxim** [cvg/nicer-slam#20]
3 occurences of
`torchvision.utils.make_grid(normal_maps_plot, scale_each=False, normalize=False, ncolumn=plot_nrow)`
in the file utilized in scene training:
https://github.com/cvg/nicer-slam/blob/main/code/utils/plots.py

`ncolumn` does not exist in make_grid(...) but `nrow` does, see https://docs.pytorch.org/vision/main/generated/torchvision.utils.make_grid.html

Hence, the code can not be executed. Please look into it.
